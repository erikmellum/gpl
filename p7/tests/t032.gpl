int nums[4];

on akey
{
    nums[0] = 42;
}

on skey
{
    print("nums[0] = " + nums[0]);
}

