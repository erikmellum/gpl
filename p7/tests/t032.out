gpl.cpp::main()
  input file(tests/t032.gpl)
  random seed(42)
  read_keypresses_from_standard_input(true)
  dump_pixels(false)
  symbol_table(true)
  print_symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
int nums[0] 0
int nums[1] 0
int nums[2] 0
int nums[3] 0
gpl.cpp::main() Calling window->initialize().
gpl.cpp::main() Passing control to window->main_loop().
gpl[10]: nums[0] = 42

Printing the symbol table from signal_handler()
int nums[0] 42
int nums[1] 0
int nums[2] 0
int nums[3] 0
